### ISSUE TITLE
About the test accuracy

### ISSUE THREAD

**Issue by jewelc92** [xuannianz/keras-fcos#16]
I train on pascal 'trainval.txt' with 5011 images, val on 'test.txt' with 4952 images, after 50 epochs the validation mAP is 0.33, while train with keras-retinanet, after 35 epochs the validation mAP is 0.63，I don't know why
